docs/data/tree-view/rich-tree-view/editing/editing.md

```diff
@@ -0,0 +1,98 @@
+---
+productId: x-tree-view
+title: Rich Tree View - Editing
+githubLabel: 'component: tree view'
+packageName: '@mui/x-tree-view'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
+packageName: '@mui/x-tree-view'
+---
+
+# Rich Tree View - Label editing
+
+<p class="description">Learn how to edit the label of Tree View items.</p>
+
+## Enable label editing
+
+You can use the `isItemEditable` prop to enable editing on certain items.
```

Comment:
I would remove the `:::success` and just show a 1st demo with all items enabled and then have a section `# Limit editing to some items` or something like that (or in the same section if you prefer)

Having the most basic example in a `:::success` is not optimal IMHO.

Something like that:

```
## Enable label editing

You can use the `isItemEditable` prop to enable editing.
If set to `true`, this prop will enable label editing on all items:

{{"demo": "LabelEditingAllItems.js"}}

If you pass a method to `isItemEditable`, only the items for which the method returns `true` will be editable:

{{"demo": "LabelEditingSomeItems.js"}}

:::success
If an item is editable, the editing state can ........
:::
```

I moved the UX description in a `:::success` because for me it is out of the main goal of the section (which is how to enable the feature).
If you prefer you can create a new section, but I don't think it's worth it

Reply:
Done! I made some slight changes to your suggestion though:
- `Enable editing` is its own section, and the success callout below the first demo describes the UX/interactions
- `Limit editing to some items` is the second section, and `Limit editing to leaves` is an h3 within that section

WDYT?

Reply:
Looks good to me :+1: